rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -202,17 +203,20 @@ void getLockWhitelistSize() {
         @Test
         void getLockWhitelistEntryByIndex() {
             LockWhitelistEntry entry = mock(LockWhitelistEntry.class);
-            when(whitelistSupport.getLockWhitelistEntryByIndex(0)).thenReturn(entry);
+            when(whitelistSupport.getLockWhitelistEntryByIndex(0)).thenReturn(Optional.of(entry));
 
-            assertEquals(entry, bridgeSupport.getLockWhitelistEntryByIndex(0));
+            Optional<LockWhitelistEntry> lockWhitelistEntryByIndex = bridgeSupport.getLockWhitelistEntryByIndex(0);
+            assertTrue(lockWhitelistEntryByIndex.isPresent());
+            assertEquals(entry, lockWhitelistEntryByIndex.get());
         }
 
         @Test
         void getLockWhitelistEntryByAddress() {
             LockWhitelistEntry entry = mock(LockWhitelistEntry.class);
-            when(whitelistSupport.getLockWhitelistEntryByAddress("address")).thenReturn(entry);
+            when(whitelistSupport.getLockWhitelistEntryByAddress("address")).thenReturn(Optional.of(entry));
 
-            assertEquals(entry, bridgeSupport.getLockWhitelistEntryByAddress("address"));
+            assertTrue(bridgeSupport.getLockWhitelistEntryByAddress("address").isPresent());
+            assertEquals(entry, bridgeSupport.getLockWhitelistEntryByAddress("address").get());
```

Comment:
Nit, but I would do the call to bridgeSupport only once. Store the result in a variable and then do the assertions with that variable